lib/instance_agent/plugins/codedeploy/command_executor.rb

```diff
@@ -473,6 +473,24 @@ def app_spec_path
           'appspec.yml'
         end
 
+        # Checks for existence the possible extensions of the app_spec_path (.yml and .yaml)
+        private
+        def app_spec_real_path(deployment_spec)
+          app_spec_yaml_location = File.join(archive_root_dir(deployment_spec), "appspec.yaml")
+          app_spec_yml_location = File.join(archive_root_dir(deployment_spec), "appspec.yml")
+          app_spec_default_location = File.join(archive_root_dir(deployment_spec), deployment_spec.app_spec_path)
+          if File.exist? app_spec_default_location
+            log(:debug, "BBBBB Using appspec file #{app_spec_default_location}")
```

Comment:
No clue what "BBBBB" is

Reply:
Good catch! I think my keyboard was stuck or something. Will remove it.